libs/orderbook/src/lib/orderbook.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+
+import Orderbook from './orderbook';
+
+describe('Orderbook', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<Orderbook data={null} decimalPlaces={4} />);
```

Comment:
I did not added any more module tests, because I expect changes in data presentation layer. Everything else is covered by unit tests.

Reply:
Fine with me